Automattic/wp-calypso
# Plans: Migrate images to webpack

#### Changes proposed in this Pull Request

* Migrates all Plans-page related images to webpack to fix #34634 and #34641 
* Some of these images are used in other areas of Calypso, so I've migrated those instances as well.
* Hashed images in the source look like this:

<img width="481" alt="Screen Shot 2019-07-25 at 3 24 20 PM" src="https://user-images.githubusercontent.com/2124984/61958531-8a7ad280-af8f-11e9-985c-80947ff67317.png">

#### Testing instructions

* Switch to this PR. Check the source of all illustrations on the following sections to ensure they've been migrated properly; they'll have a hash at the end of the URL like the one in the screenshot above:

* Plans page (`/my-plan`) for all WP.com plan types (Blogger, Personal, Premium, Business, eCommerce)
* Jetpack Plans page (`/my-plan`) for all Jetpack plan types (Personal, Premium, Professional)
* Checkout thank-you screen for all WP.com and Jetpack plan types
* "Use your own domain" images under `/domains/add` and in signup under `/start/domains`
* Jetpack Activity Log - image in the intro banner at the top of the page
